projects/kit/components/input-month/native-month-picker/native-month-picker.style.less

```diff
@@ -0,0 +1,17 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+// TODO: add tui-textfield:has([tuiInputMonth])
+tui-textfield input[type='month'] {
+    .fullsize();
+
+    opacity: 0;
+    border: 0;
+
+    &::-webkit-calendar-picker-indicator {
+        .fullsize(); // Extends clickable area for icon to open native calendar
+    }
+}
+
+tui-textfield:has([tuiInputMonth]) .t-clear {
+    z-index: 1;
```

Comment:
Does it make any sense for clear not to be above everything else out of the box? Maybe we can just apply this rule all the time

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/10345/commits/57b3855e2c9dba9441bc6bc2f5db2365dc45139e